app/backend/src/clj/teet/meeting/meeting_pdf.clj

```diff
@@ -0,0 +1,428 @@
+    [teet.localization :refer [with-language tr tr-enum]]
+    [teet.user.user-model :as user-model])
+  (:import (com.vladsch.flexmark.parser Parser)
+           (com.vladsch.flexmark.util.ast Document)
+           (com.vladsch.flexmark.ast
+            ;; Import node types for rendering
+            Paragraph BulletList OrderedList Heading Text
+            StrongEmphasis Emphasis)))
+
+(declare render-md)
+
+(def default-layout-config
+  {;; A4 portrait width/height
+   :width "21cm"
+   :height "29.7cm"
+
+   ;; 1cm margins
+   :margin-top "1cm" :margin-bottom "1cm"
+   :margin-left "1cm" :margin-right "1cm"
+
+   :body {:margin-top "1cm"}
+   :header {:extent "1cm"}
+   :footer {:extent "1cm"}})
+
+(def link-look-and-feel
+  {:font-size "16px"
+   :border-top-style "solid"
+   :border-top-width "1"
+   :border-top-color "#D2D3D8"
+   :padding-top 8
+   :padding-bottom 10})
+
+(def border-1-mm
+  {:border-top-style "solid"
+   :border-top-width "1"
+   :border-top-color "#D2D3D8"})
+
+(def reviewers-look-and-feel
+  {:font-size "14px"
+   :padding-top 9
+   :padding-bottom 12})
+
+(def reviewers-date-look-and-feel
+  {:font-size "14px"
+   :text-align "right"
+   :padding-top 9
+   :padding-bottom 12})
+
+(def date-format
+  (doto (java.text.SimpleDateFormat. "MM.dd.yyyy" )
+    (.setTimeZone (java.util.TimeZone/getTimeZone "Europe/Tallinn"))))
+
+(def time-format
+  (doto (java.text.SimpleDateFormat. "HH:mm" )
+    (.setTimeZone (java.util.TimeZone/getTimeZone "Europe/Tallinn"))))
+
+(def time-sec-format
+  (doto (java.text.SimpleDateFormat. "HH:mm:ss" )
+    (.setTimeZone (java.util.TimeZone/getTimeZone "Europe/Tallinn"))))
+
+(defn format-date
+  "Format date in human readable locale specific format, eg. dd.MM.yyyy"
+  [date]
+  (when date
+    (.format date-format date)))
+
+(defn format-time
+  "Format time with minute resolution"
+  [date]
+  (when date
+    (.format time-format date)))
+
+(defn format-time-sec
+  "Format time with seconds resolution"
+  [date]
+  (when date
+    (.format time-sec-format date))
+  )
+
+(defn- meeting-time
+  "Format meeting begin, end time, date"
+  [meeting]
+  (str (format-date (:meeting/start meeting))
+       " " (format-time (:meeting/start meeting)) " - "
+       (format-time (:meeting/end meeting))))
+
+(defn- approval-date-time
+  "Format approval review date and time"
+  [review]
+  (str (format-date review) " " (format-time-sec review)))
+
+(defn- decision-list-item
+  "Return the agenda topic descition list item"
+  [decision topic]
+  (let [title (str topic " " (tr+ [:fields :meeting.decision/body]) " #" (:meeting.decision/number decision))
+        decision-text (render-md (:meeting.decision/body decision))]
+    [:fo:block
+     [:fo:block {:font-size "24px" :font-weight "400"} title]
+     [:fo:block {:font-size "14px" :font-weight "400" :space-after "40"} decision-text]]))
+
+(defmulti link-list-item (fn [link meeting project-id] (:link/type link)))
```

Comment:
Fix unused bindings, by linting convention add "_" prefix to something that we want in the signature but don't care about in this context. So `_meeting`